common/edicts/00_edicts.txt

```diff
@@ -0,0 +1,1107 @@
+##########################################################################
+# Empire edicts
+##########################################################################
+
+@Edict1Cost = 10
+@Edict2Cost = 20
+@Edict3Cost = 30
+
+@EdictPerpetual = -1
+
+#################
+# Toggle Edicts
+#################
+evacuation_protocols = {
```

Comment:
If we're going the Doomsday route for the Rakhana (I'm fairly certain that was the decision) we should repurpose this to use their origin.